itou/users/tests/test_management_commands.py

```diff
@@ -33,19 +33,19 @@ def test_deduplicate_job_seekers(self):
         }
 
         # Create `user1`.
-        job_app1 = JobApplicationFactory(with_approval=True, job_seeker__nir=None, **kwargs)
+        job_app1 = JobApplicationFactory(with_approval=True, job_seeker__nir="", **kwargs)
         user1 = job_app1.job_seeker
 
-        assert user1.nir is None
+        assert not user1.nir
```

Comment:
L’assertion est plus précise. (répété plusieurs fois dans la PR)
```suggestion
        assert user1.nir == ""
```